### ISSUE TITLE
does not start on ubuntu

### ISSUE THREAD

**Issue by teo2022** [ducha-aiki/mods#21]
I installed everything according to the instructions

root@80-78-247-58:/var/www/html/mods# ./mods.cpp examples/cat.png examples/1.png 3.png 2.png k1.txt k2.txt m.txt l.txt 0 0 examples/cat.txt config_iter_mods_cviu.ini iters_mods_cviu.ini
./mods.cpp: line 1: /*------------------------------------------------------*/: No such file or directory
./mods.cpp: line 2: /bin: Is a directory
./mods.cpp: line 3: /*------------------------------------------------------*/: No such file or directory
./mods.cpp: line 33: //#define: No such file or directory
./mods.cpp: line 47: using: command not found
./mods.cpp: line 49: const: command not found
./mods.cpp: line 49: //number: No such file or directory
./mods.cpp: line 51: syntax error near unexpected token `('
./mods.cpp: line 51: `//inline long getMilliSecs()'
root@80-78-247-58:/var/www/html/mods#         

root@80-78-247-58:/var/www/html/mods# ./mods examples/cat.png examples/1.png out1.png out2.png k1.txt k2.txt m.txt l.txt 0 0 examples/cat.txt config_iter_mods_cviu.ini iters_mods_cviu.ini
-bash: ./mods: No such file or directory

**Comment by ducha-aiki** [comment]
First, I don't recommend to run it under root.
Second, you cannot run cpp file, you need to compile everything first.

**Comment by teo2022** [comment]
I followed these commands! still the same (do you need to do something else?

cd vlfeat
make

cd ../build
cmake ..
make

**Comment by teo2022** [comment]
root@80-78-247-58:/var/www/html/mods/build# cmake ..
Looking for OpenCV
Not found!
-- A library with BLAS API found.
-- A library with LAPACK API found.
OpenMP found
OPENMP FOUND
-- Configuring done
-- Generating done
-- Build files have been written to: /var/www/html/mods/build

**Comment by ducha-aiki** [comment]
OK, then you should go into build directory and run the compiled file, not .cpp

./mods examples/cat.png examples/cat2.png out1.png out2.png k1.txt k2.txt m.txt l.txt 0 0 examples/cat.txt config_iter_mods_cviu.ini iters_mods_cviu.ini

**Comment by teo2022** [comment]
-bash: ./mods: No such file or directory

**Comment by ducha-aiki** [comment]
Then you haven't compiled it (I don't see logs from running make command), or you are not in the correct directory

**Comment by teo2022** [comment]
<img width="1043" alt="Снимок экрана 2021-05-06 в 17 56 05" src="https://user-images.githubusercontent.com/21157789/117287233-576baf80-ae94-11eb-826a-7a505f26c0bb.png">

**Comment by ducha-aiki** [comment]
Of course, there is no mods file, there is "fatal error".
 Another hint: when you run cmake command, it didn't find OpenCV.
So, please, install OpenCV first.

**Comment by teo2022** [comment]
<img width="758" alt="Снимок экрана 2021-05-06 в 19 20 06" src="https://user-images.githubusercontent.com/21157789/117297147-124d7a80-aea0-11eb-8c97-92f11cdda1b7.png">

**Comment by teo2022** [comment]
<img width="1002" alt="Снимок экрана 2021-05-06 в 19 20 22" src="https://user-images.githubusercontent.com/21157789/117297177-1d080f80-aea0-11eb-8bb6-c8560ad038d2.png">

**Comment by teo2022** [comment]
can't install this package, already tried different options

<img width="718" alt="Снимок экрана 2021-05-06 в 19 21 23" src="https://user-images.githubusercontent.com/21157789/117297298-4032bf00-aea0-11eb-82a7-63cf34459c6a.png">

**Comment by teo2022** [comment]
ubuntu 20.04

**Comment by teo2022** [comment]
sudo apt install python3-opencv     - installed this package

**Comment by ducha-aiki** [comment]
The instructions are for ubuntu 14.04, which is stated in the readme. 
Sorry, but I cannot guide you on how to install OpenCV on modern Ubuntu, there are enough resources on the web on it, and also C++ development is tricky and might be time-consuming.

**Comment by teo2022** [comment]
i installed it, here is the screen
<img width="758" alt="117297147-124d7a80-aea0-11eb-8c97-92f11cdda1b7" src="https://user-images.githubusercontent.com/21157789/117298829-02369a80-aea2-11eb-9a69-16cec13b24ed.png">

**Comment by ducha-aiki** [comment]
No, what you have installed is python version. Instead you need C++ libraries and headers, as MODS is C++ algorithm.

**Comment by teo2022** [comment]
Installed ubuntu 16.04, everything was installed according to the instructions without errors and now this is

<img width="788" alt="Снимок экрана 2021-05-06 в 19 54 35" src="https://user-images.githubusercontent.com/21157789/117301504-e680c380-aea4-11eb-8c65-e58379e998f0.png">

**Comment by teo2022** [comment]
<img width="602" alt="Снимок экрана 2021-05-06 в 19 59 13" src="https://user-images.githubusercontent.com/21157789/117302175-8b9b9c00-aea5-11eb-9391-abab36c51e9b.png">
